### DISCUSSION TITLE
GCP segment problem

### DISCUSSION THREAD

**Discussion by DataCrusade1999** [JanDeDobbeleer/oh-my-posh#2884] [Troubleshoot] [answered]
I'm trying the GCP segment of oh my posh but I get a message of **invalid template text**. I'm using the same segment that is given in the docs.

https://ohmyposh.dev/docs/segments/gcp

![image](https://user-images.githubusercontent.com/57610394/194010471-527f7757-b6ef-4897-8274-74e494aba88c.png)

I've also looked at a GCP-related issue but couldn't find the solution to my specific problem.

https://github.com/JanDeDobbeleer/oh-my-posh/issues/2814

this is what I get about the GCP segment when I run `oh-my-posh debug`

![image](https://user-images.githubusercontent.com/57610394/194011322-e28246d1-6fe7-43c1-83d2-f0ae7aa0f1f9.png)

**Comment by JanDeDobbeleer** [comment, marked as answer]
@DataCrusade1999 you're missing a bracket at `{{Data.Project}`:

```
{{ Data.Project}} :: {{ Data.Account}}
```
